test/mapwindow.jl

```diff
@@ -91,5 +91,18 @@ using ImageFiltering, Base.Test
                                          2 3 3 4 4;
                                          1 3 3 3 2]
     end
+
+    function groundtruth(f, A, window::Tuple, border, imginds)
+        mapwindow(f,A,window,border)[imginds...]
+    end
+    for (f,img, window, imginds) ∈ [
+            (mean, randn(10), (1,), (1:2:10,)),
+            (median!, randn(10), (-1:1,), (1:2:8,)),
+            (mean, randn(10), (-1:1,), (1:2:8,)),
+            (mean, randn(10,5), (-1:1,0:0), (1:2:8,1:3)),
+        ]
+        border = ImageFiltering.borderinstance("replicate")
```

Comment:
Can we not just pass `"replicate"` directly?

Reply:
Yes we can!